### PR TITLE
Add HW24.1 solution

### PR DESCRIPTION
**Pull request by OleksiiTromsa**

### CODE REVIEW

File: src/main/java/com/internet/shop/model/ShoppingCart.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.util.List;
+
+public class ShoppingCart {
+    private Long id;
+    private List<Product> products;
+    private Long userId;
+
+    public ShoppingCart(List<Product> products, Long userId) {
+        this.products = products;
```
Comment (Nokafan) [inline src/main/java/com/internet/shop/model/ShoppingCart.java]: Should be initialized by new ArrayList....

File: src/main/java/com/internet/shop/model/Order.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+public class Order {
+    private Long id;
+    private List<Product> products;
+    private Long userId;
+
+    public Order(List<Product> products, Long userId) {
+        this.products = products;
+        this.userId = userId;
+    }
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/model/Order.java]:
```suggestion
    public Order(Long userId) {
        products = new ArrayList();
        this.userId = userId;
    }
```
same in ShoppingCart

**Review by dmytro-hryhoruk — COMMENTED** [review]
Remove all daos and services except Product from this PR

File: src/main/java/com/internet/shop/model/Order.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.util.List;
+
+public class Order {
+    private Long id;
+    private List<Product> products;
+    private Long userId;
+
+    public Order(List<Product> products, Long userId) {
+        this.products = products;
```
Comment (Nokafan) [inline src/main/java/com/internet/shop/model/Order.java]: new ArrayList... it was on QnA

**Review by Nokafan — COMMENTED** [review]
Nice work

File: src/main/java/com/internet/shop/model/Order.java (hunk 2 above)
Comment (OleksiiTromsa) [reply src/main/java/com/internet/shop/model/Order.java]: Done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**